### ISSUE TITLE
Property "text-decoration-thickness" marked as mispelled

### ISSUE THREAD

**Issue by jeremyphilemon** [sasstools/sass-lint#1304]
**What version of Sass Lint are you using?**
1.13.1

**Please include any relevant parts of your configuration**
```yml
rules:
    no-misspelled-properties: 1
    ...
```

**What did you do? Please include the actual source code causing the issue.**
Added property `text-decoration-thickness`.

**What did you expect to happen?**
The property shouldn't have been marked as misspelled.

**What actually happened? Please include any error messages given to you by Sass Lint.**
It was marked as misspelled.

**If you're using a IDE plugin have you tried the CLI too?**
Yes.

---

Property reference: https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-thickness
